Fabulous.XamarinForms/samples/FabulousWeather/FabulousWeather/App.fs

```diff
@@ -0,0 +1,154 @@
+﻿namespace FabulousWeather
+
+open Fabulous
+open Fabulous.XamarinForms
+open Fabulous.XamarinForms.LiveUpdate
+open Xamarin.Forms
+
+module App =
+    type Operator = Add | Subtract | Multiply | Divide 
+
+    /// Represents a calculator button press
+    type Msg =
+        | Refresh
+
+    
+    type DataItem = {Name:string;Value:int}
```

Comment:
As a general rule for the whole file, could you try to format using the same idea than https://github.com/fsprojects/Fabulous/blob/master/Fabulous.XamarinForms/samples/Shell/Fabimals/Fabimals/Components/AnimalListPage.fs please ?